package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "event-capturing",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint"
+  },
+  "dependencies": {
+    "@vue/composition-api": "^1.0.3",
+    "core-js": "^3.6.5",
+    "dns": "^0.2.2",
+    "secure-ls": "^1.2.6",
+    "tailwind": "^4.0.0",
+    "vue": "^3.0.0",
+    "vue-multiselect": "^1.1.5",
+    "vue-router": "^4.0.10",
+    "vue-select": "^3.11.2",
```

Comment:
Are you using all three

`vue-multiselect`, `vue-select`, and `vue-simple-search-dropdown`? 

Can you manage with only one of them?

Reply:
Actually, sorry, i was just trying them out, I don't use any of them. I'll be removing them.